OpenRA.Mods.Common/Traits/OrderProvider.cs

```diff
@@ -0,0 +1,51 @@
+﻿using OpenRA.Traits;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	// TODO: We don't need this interface if we could find multiples of traits on actors.
```

Comment:
What's going on here?  This works - we do it in many times through the engine and common mod code.